corgidrp/data.py

```diff
@@ -1046,6 +1046,132 @@ def save(self, filedir=None, filename=None):
         hdulist.writeto(self.filepath, overwrite=True)
         hdulist.close()
 
+class SpecFilterOffset(Image):
+    """
+    calibration class that contains a dictionary of the x/y offsets of the different used filters.
+
+    Args:
+        data_or_filepath (str or dict): either the filepath to the FITS file to read in OR the dictionary containing 
+                                        the filter offsets in pixel units as a paired list [&x, &y]
```

Comment:
should we mention to look at `SpecFIlterOffset.default_offsets as an example of the format?